fdbcli/fdbcli.actor.cpp

```diff
@@ -2056,6 +2062,25 @@ ACTOR Future<bool> exclude( Database db, std::vector<StringRef> tokens, Referenc
 		}
 
 		if(!force) {
+			if (markFailed) {
+				state bool safe;
+				try {
+					bool _safe = wait(makeInterruptable(checkSafeExclusions(db, addresses)));
```

Comment:
the `wait` statement requires a type declaration on the LHS, but if we declare the state variable inside the try block, it would be out of scope when we try to use it